src/plugins/workspace/public/components/workspace_form/workspace_detail_form_details.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiSuperSelect,
+  EuiColorPicker,
+  EuiCompressedFormRow,
+  EuiDescribedFormGroup,
+  EuiCompressedTextArea,
+  EuiCompressedFieldText,
+} from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { useObservable } from 'react-use';
+import {
+  detailsName,
+  detailsColorLabel,
+  detailsNameHelpText,
+  detailsUseCaseLabel,
+  detailsColorHelpText,
+  detailsNamePlaceholder,
+  detailsDescriptionPlaceholder,
+  detailsDescriptionIntroduction,
+} from './constants';
+import { CoreStart } from '../../../../../core/public';
+import { getFirstUseCaseOfFeatureConfigs } from '../../utils';
+import { DEFAULT_NAV_GROUPS } from '../../../../../core/public';
+import { useWorkspaceFormContext } from './workspace_form_context';
+import { WorkspaceUseCase as WorkspaceUseCaseObject } from '../../types';
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+
+interface WorkspaceDetailFormDetailsProps {
+  availableUseCases: Array<
+    Pick<WorkspaceUseCaseObject, 'id' | 'title' | 'description' | 'systematic'>
+  >;
+}
+
+export const WorkspaceDetailFormDetailsProps = ({
+  availableUseCases,
+}: WorkspaceDetailFormDetailsProps) => {
+  const {
+    formData,
+    isEditing,
+    formErrors,
+    handleColorChange,
+    handleUseCaseChange,
+    handleNameInputChange,
+    handleDescriptionChange,
+  } = useWorkspaceFormContext();
+  const {
+    services: { workspaces },
+  } = useOpenSearchDashboards<CoreStart>();
+  const [value, setValue] = useState(formData.useCase);
```

Comment:
Why should it store `formData.useCase` as component local state? Cann't it just rely on `formData.useCase`?

Reply:
The aim is to reset the useCase select table value when user modify the useCase value and click the discard changes button.